### PR TITLE
Adding a more interactive readme file

### PR DESCRIPTION
**Pull request by Shivamishra21**
I have added a different markdown file that is more interactive from the older one. It contains details about installation and contribution too. Please check it and merge to your master branch if you are satisfied with the changes.

### CODE REVIEW

File: attractiveReadme.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<h1 align = "center"> SumIt</h1>
+
+## Overview
+---
```
Comment (annu12340) [inline attractiveReadme.md]: Could you remove the ```---``` everywhere

File: attractiveReadme.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+python -m flask run
+```
+
+* Copy the localhost url from terminal and paste in your browser.
+## Contribution
+---
+
+Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are ***highly appreciated***.
+
+ *   Fork the Proje
```
Comment (annu12340) [inline attractiveReadme.md]: ```Proje``` typo

File: attractiveReadme.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+So  here is ***SumIt***  an intelligent text summarizer that creates a coherent and short summary of all information that were discussed in seminars/workshops/meetings etc.
+
+## How to use  SumIt
+---
+It is a text summarizer that summarise content.
+
+It is vey simple to use.....
+
+* Firstly just type the text inside the text area provided.
+* Then n finally click the button marked as "Generate".
```
Comment (annu12340) [inline attractiveReadme.md]: ```Then n ``` typo

**Review by annu12340 — CHANGES_REQUESTED** [review]

**Comment by Shivamishra21** [comment]
@annu12340 I have made the changes you asked for in 'attractiveReadme' branch .

**Comment by annu12340** [comment]
Everything looks good
Thanks for the pr :)

**Pull request merged**